mate-extra/mate-sensors-applet/mate-sensors-applet-1.24.1.ebuild

```diff
@@ -43,6 +43,11 @@ DEPEND="${COMMON_DEPEND}
 
 PDEPEND="hddtemp? ( dbus? ( sys-fs/udisks:2 ) )"
 
+src_prepare() {
+    eapply "${FILESDIR}"/${PN}-1.24.1-dont-check-for-headers.patch
```

Comment:
If no eapply is already being used, use PATCHES in global scope (above src_prepare).

Then you don't need to define src_prepare at all.